$NetBSD$

	Setup NetBSD parameters
	Due to current lack of XDR quad marshalling NDMP
	need to be disabled.

--- core/CMakeLists.txt.orig	2019-02-13 14:25:55.000000000 +0000
+++ core/CMakeLists.txt
@@ -119,6 +119,16 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "FreeBS
    link_libraries(intl)
 endif()
 
+ 
+if (${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
+   set(HAVE_NETBSD_OS 1)
+   set(ndmp 0)
+   set(PSCMD "ps -ax -o pid,command")
+   include_directories(/usr/pkg/include)
+   link_directories(/usr/pkg/lib)
+   link_libraries(intl)
+endif()
+
 IF (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
    set(HAVE_DARWIN_OS 1)
    #set(CMAKE_INCLUDE_PATH  "${OPENSSL_INCLUDE_DIR}:${CMAKE_INCLUDE_PATH}")
@@ -477,6 +487,7 @@ if ("${HAVE_CAPABILITY}")
    message( STATUS "CAPABILITY set")
 endif()
 
+set(HAVE_NDMP 0)
 set(ndmp "${ndmp}" )
 set(lmdb "${lmdb}" )
 set(build_client_only "${client-only}" )
